frontend/mulighetsrommet-api-client/src/models/Tiltaksgjennomforing.ts

```diff
@@ -2,14 +2,28 @@
 /* tslint:disable */
 /* eslint-disable */
 
-import type { Tiltakskode } from './Tiltakskode';
+import { Tiltakstype } from "./Tiltakstype";
+import { Arrangor } from "./Arrangor";
+import { Tiltaksansvarlig } from "./Tiltaksansvarlig";
 
 export type Tiltaksgjennomforing = {
-    id: number;
-    tittel: string;
-    beskrivelse: string;
-    tiltakskode: Tiltakskode;
-    tiltaksnummer?: string;
-    fraDato: string | null;
-    tilDato: string | null;
+  _id: number;
+  tiltakstype: Tiltakstype;
+  tiltaksgjennomforingNavn: string;
+  beskrivelse?: string;
+  tiltaksnummer: number;
+  kontaktinfoArrangor: Arrangor;
+  lokasjon: string;
+  enheter: { fylke: string };
+  oppstart: string;
+  oppstartsdato?: Date;
+  faneinnhold?: {
+    forHvemInfoboks?: string;
+    forHvem?: object;
+    detaljerOgInnholdInfoboks?: string;
+    detaljerOgInnhold?: object;
+    pameldingOgVarighetInfoboks?: string;
+    pameldingOgVarighet?: object;
```

Comment:
Kan du type opp disse propertyene som har type `object`? Når de _bare_ er av typen `object` så er det ikke så lett å jobbe med de videre i koden. Du kan lage en type og gjenbruke den for feltene antar jeg.

Reply:
Hmm, ja da trenger jeg litt hjelp, for det er de feltene fra Sanity som er av typen `blockContent`. Fant ikke helt ut av det mens jeg holdt på. Er noen steder jeg har satt de til `any` også, fordi typen `TypedObject` ikke funket

Reply:
Du kan bruke disse i første omgang, så ser vi hvordan det funker
```
interface BlockContent {
  _key: string;
  _type: string;
  children: Child[];
  markDefs: any[];
  style: string;
}

interface Child {
  _key: string;
  _type: string;
  marks: any[];
  text: string;
}
```

For å få de så tok jeg utgangspunkt i en fetch-request, så kopierte jeg ut json til vscode og brukte extensionen https://marketplace.visualstudio.com/items?itemName=MariusAlchimavicius.json-to-ts for å konvertere til typescript. 

Jeg tenker det ikke er så krise at `marks` er av typen `any`. Det vi er interessert i per nå er `text`-propertyen fra `Child`.